go/tasks/plugins/webapi/bigquery/plugin.go

```diff
@@ -0,0 +1,503 @@
+package bigquery
```

Comment:
shall we add a comment above the package name. I know this is not done everywhere, but we want to start doing this so that we get automated documentation